index.html

```diff
@@ -31,5 +31,149 @@ <h1>I’m Victor Ozoede Glad to see you!</h1>
+    </section>
+    <!-- First section ends -->
+    <!-- Second section starts -->
+    <section class="work-one work">
+      <img class ="img-work" src="img/Portfolio2.svg" alt="Tonic">
+      <h2>Tonic</h2>
+        <ul class="work-top">
+          <li class="canopy">CANOPY</li>
+          <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+          <li class="dev">back end dev</li>
+          <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+          <li class="dev">2015</li>
+        </ul>
+        <p class="body-paragraph">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+          <ul class="work-bottom">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+        <form action="#">
+            <button class="btn-body" type="submit" name="submit">see project</button>
+        </form>
+    </section>
+    <!-- Second Section ends -->
+    <!-- Third section starts -->
+    <section class="work-one work">
+      <img class ="img-work" src="img/Portfolio3.svg" alt="Tonic">
+      <h2>Tonic</h2>
+        <ul class="work-top">
+          <li class="canopy">CANOPY</li>
+          <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+          <li class="dev">back end dev</li>
+          <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+          <li class="dev">2015</li>
+        </ul>
+        <p class="body-paragraph">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+          <ul class="work-bottom">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+        <form action="#">
+            <button class="btn-body" type="submit" name="submit">see project</button>
+        </form>
+    </section>
+    <!-- Third section ends -->
+    <!-- Fourth section starts -->
+    <section class="work-one work">
+      <img class ="img-work" src="img/Portfolio4.svg" alt="Tonic">
+      <h2>Tonic</h2>
+        <ul class="work-top">
+          <li class="canopy">CANOPY</li>
+          <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+          <li class="dev">back end dev</li>
+          <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+          <li class="dev">2015</li>
+        </ul>
+        <p class="body-paragraph">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+          <ul class="work-bottom">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+        <form action="#">
+            <button class="btn-body" type="submit" name="submit">see project</button>
+        </form>
+    </section>
+    <!-- Fourth section ends -->
+    <!-- Body introduction section starts -->
+    <section class="about-myself">
```

Comment:
[OPTIONAL]
- Keep your code clean and try to use comments unless if you have something important to keep. While you are coding. After that, clean your files and send it to review.